packages/adapter-triplit/tests/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Start the server in the background
+pnpm exec triplit dev &
+SERVER_PID=$!
+
+# Optional: Wait for the server to be ready
+sleep 2
+
+# Run migration
+pnpm exec triplit migrate up --token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ4LXRyaXBsaXQtdG9rZW4tdHlwZSI6InNlY3JldCIsIngtdHJpcGxpdC1wcm9qZWN0LWlkIjoibG9jYWwtcHJvamVjdC1pZCJ9.8Z76XXPc9esdlZb2b7NDC7IVajNXKc4eVcPsO7Ve0ug
```

Comment:
What exactly is this hardcoded token for? 

Is there a way to run triplit locally, via a docker conatiner perhaps, to run tests against?

Many database service providers offer a local version of their database for folks to run tests against, develop against, etc. Many of our adapters leverage this to avoid having to use cloud service provider keys in our repository for tests, etc.

Reply:
`triplit dev` will start a node server on your machine that runs the database, so everything is running locally. The command will also generate and output a service token for running admin commands on the server. The schema of your local database at the moment can only be updated via migrations, which (even locally) require some admin token. It's similar to Supabase's test/local dev set up, but I hard coded the token that the `dev` command would return instead of trying to find it in the output (which I believe the supabase script does). Happy to set up a pattern like that if you'd like though.

We actually have some stuff going in shortly for local dev that will let you pass in a schema file directly to a dev server (so no need to run this admin command), so can update this when thats in as well.

Reply:
Ah okay, so it is a local instance and this is the hardcoded credential for that only? 

That's okay with me then 👍

Reply:
@ndom91 Yea exactly 👍

Going to rebase to fixup the lockfile.